File: dataLayer/entities/recognition_image.py
from sqlalchemy import Column, String, Integer, Date, ForeignKey

from dataLayer.database_connection import Base


class RecognitionImage(Base):
    __tablename__ = 'RecognitionImage'

    id = Column(Integer, primary_key=True)
    name = Column('Name', String)
    thumbnail = Column("Thumbnail", String)
    recognition_id = Column("RecognitionId", Integer, ForeignKey("Recognition.Id"))

    def __init__(self, name, recognition_id):
        self.name = name
        self.recognition_id = recognition_id

File: DataLayer/Entities/RecognitionImage.cs
using System.IO;
using DataLayer.Entities.Common;

namespace DataLayer.Entities
{
    public class RecognitionImage : EntityBase, IImage
    {
        public int? RecognitionId { get; set; }
        public Recognition Recognition { get; set; }
        public string Name { get; set; }
        public string Thumbnail { get; set; }
        public string Url { get; set; }
        public string GetPath()
        {
            return $"{nameof(RecognitionImage)}/{Id}";
        }
    }
}